apps/prairielearn/assets/scripts/question.ts

```diff
@@ -164,3 +170,58 @@ function updateStatus(submission) {
   }
   display.innerHTML = label;
 }
+
+function setupDynamicObjects() {
+  // Install on page load and reinstall on websocket re-render
```

Comment:
Not necessarily requesting a change, just sharing information: in PrairieTest, we've been using [`selector-observer`](https://github.com/rkusa/selector-observer) for this kind of thing where we want to do something for every element matching a selector, regardless of if it exists now or is added at some point in the future.

Reply:
I'd prefer not to change this now, but noted.